addons/stock_picking_wave/stock_picking_wave.py

```diff
@@ -56,11 +57,19 @@ def done(self, cr, uid, ids, context=None):
                     continue
                 if picking.state != 'assigned':
                     raise UserError(_('Some pickings are still waiting for goods. Please check or force their availability before setting this wave to done.'))
+                message_body = "<b>%s:</b> %s <a href=#id=%s&view_type=form&model=stock.picking.wave>%s</a>" % (_("Transferred by"), _("Picking Wave"), wave.id, wave.name)
+                picking.message_post(body=message_body)
                 picking_todo.add(picking.id)
         if picking_todo:
             self.pool.get('stock.picking').action_done(cr, uid, list(picking_todo), context=context)
         return self.write(cr, uid, ids, {'state': 'done'}, context=context)
 
+    def _track_subtype(self, cr, uid, ids, init_values, context=None):
+        record = self.browse(cr, uid, ids[0], context=context)
+        if 'state' in init_values and record.state != 'draft':
```

Comment:
Well that's a side-effect of the feature ... it will be improved a lot in a near future. So I would simplify the code and remove the condition, and see how it is displayed once all parts of the chatter feature are merged.